de4dot.code/deobfuscators/Dotfuscator/StringDecrypter.cs

```diff
@@ -74,7 +76,7 @@ public class StringDecrypterInfo {
 
 				simpleDeobfuscator.Deobfuscate(method);
 				var instrs = method.Body.Instructions;
-				for (int i = 0; i < instrs.Count - 3; i++) {
+				for (int i = 0; i < instrs.Count - 6; i++) {
```

Comment:
This can be restored to -3 again since the code below already has the -6 test

Reply:
ok.